dockstore-webservice/src/main/resources/migrations.xml

```diff
@@ -27,6 +27,8 @@
     <include file="migrations.1.6.0.xml" relativeToChangelogFile="true"/>
     <include file="migrations.1.7.0.xml" relativeToChangelogFile="true"/>
     <include file="migrations.1.7.0.relinquish.xml" relativeToChangelogFile="true"/>
+    <include file="migrations.test.add_service_1.7.0.xml" relativeToChangelogFile="true"/>
```

Comment:
If this is the case, should the test migrations below be fitted into the correct order as well? 
Some might be hard to figure out, but it would seem that the 1.5.0 migrations should be between 1.5.0 and 1.6.0?

Reply:
I updated the order based on some examples I found in the CommonTestUtilities.java file. Hopefully it just works.